apps/shared/avifjpeg.c

```diff
@@ -354,24 +380,133 @@ avifBool avifJPEGRead(const char * inputFilename,
         }
     }
     if (!ignoreXMP) {
-        const avifROData tagStandardXmp = { (const uint8_t *)"http://ns.adobe.com/xap/1.0/\0", 29 };
-        const avifROData tagExtendedXmp = { (const uint8_t *)"http://ns.adobe.com/xmp/extension/\0", 35 };
-        avifBool found = AVIF_FALSE;
+        const avifROData tagStandardXMP = { (const uint8_t *)"http://ns.adobe.com/xap/1.0/\0", 29 };
+        avifBool foundStandardXMP = AVIF_FALSE;
```

Comment:
Nit/optional: `foundStandardXMP` can be omitted. We can just test if `standardXMPData` is non-null.